kpi/tests/api/v2/test_api_assets.py

```diff
@@ -154,10 +140,12 @@ def test_restricted_access_to_version(self):
         self.assertEqual(resp2.data['detail'], 'Not found.')
 
 
-class AssetsDetailApiTests(BaseTestCase):
+class AssetsDetailApiTests(BaseAssetTestCase):
     fixtures = ['test_data']
 
-    URL_NAMESPACE = ROUTER_URL_NAMESPACE
+    def __init__(self, *args, **kwargs):
+        self.URL_NAMESPACE = ROUTER_URL_NAMESPACE
```

Comment:
What is the need to add `URL_NAMESPACE` to instance's context?
Not that I'm against it but it's strange to have only this class using it this way. 
A wise man told me one day, we had to be consistent 😏

Reply:
Aha, yes. I was hoping you'd comment on this. I knew either I was confused or something was off about the `URL_NAMESPACE` stuff overall; I'm glad it turned out to be the former :)